draft-ietf-oauth-sd-jwt-vc.md

```diff
@@ -355,14 +355,12 @@ be ignored by the Verifier.
 
 ## Examples
 
-The following is a non-normative example of a presentation of the SD-JWT shown
-above including a Key Binding JWT:
+The following is a non-normative example of a presentation of the SD-JWT shown in Section 3.3.
+In this presentation, the Holder provides only the Disclosure for the claim `address`
+including a Key Binding JWT. Other claims are not disclosed to the Verifier:
```

Comment:
Thanks Lukas! 
The suggestion below has a few editorail updates and changes the way the section 3.3 reference works. 

```suggestion
The following is a non-normative example of a presentation of the SD-JWT shown in (#example) including a Key Binding JWT.
In this presentation, the Holder provides only the Disclosure for the  `address` claim. 
Other claims are not disclosed to the Verifier.
```

Reply:
@bc-pi I updated with the link to reference section 3.3 :)